src/commands/cloud-run/flare.ts

```diff
@@ -4,27 +4,43 @@ import path from 'path'
 import process from 'process'
 import util from 'util'
 
+import {Logging} from '@google-cloud/logging-min'
 import {ServicesClient} from '@google-cloud/run'
 import {google} from '@google-cloud/run/build/protos/protos'
 import chalk from 'chalk'
 import {Command} from 'clipanion'
 import {GoogleAuth} from 'google-auth-library'
 
-import {API_KEY_ENV_VAR, CI_API_KEY_ENV_VAR, FLARE_OUTPUT_DIRECTORY} from '../../constants'
+import {API_KEY_ENV_VAR, CI_API_KEY_ENV_VAR, FLARE_OUTPUT_DIRECTORY, LOGS_DIRECTORY} from '../../constants'
 import {sendToDatadog} from '../../helpers/flare'
 import {createDirectories, deleteFolder, writeFile, zipContents} from '../../helpers/fs'
 import {requestConfirmation} from '../../helpers/prompt'
 import * as helpersRenderer from '../../helpers/renderer'
-import {maskString} from '../../helpers/utils'
+import {formatBytes, maskString} from '../../helpers/utils'
 
 import {SKIP_MASKING_CLOUDRUN_ENV_VARS} from './constants'
+import {CloudRunLog, LogConfig} from './interfaces'
 import {renderAuthenticationInstructions} from './renderer'
 
 const SERVICE_CONFIG_FILE_NAME = 'service_config.json'
 const FLARE_ZIP_FILE_NAME = 'cloudrun-flare-output.zip'
+const ALL_LOGS_FILE_NAME = 'all_logs.csv'
+const TEXT_LOGS_FILE_NAME = 'text_logs.csv'
+const WARNING_LOGS_FILE_NAME = 'warning_logs.csv'
+const ERRORS_LOGS_FILE_NAME = 'error_logs.csv'
+const DEBUG_LOGS_FILE_NAME = 'DEBUG_logs.csv'
+
+// Must be in range 0 - 1000
+export const MAX_LOGS_PER_PAGE = 1000
+// There will be a maximum of (MAX_LOGS_PER_PAGE * MAX_PAGES) logs for each log file
+// The more pages there are, the longer the program will take to run
+export const MAX_PAGES = 3
```

Comment:
Do we need more than 1000 logs per file? Or is the pagination overkill?

Reply:
Maybe ask to Support?

Reply:
We only need 1000 logs per file, so I removed pagination and we get a max of 1000 logs now